App/components/SearchInput/input.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import {
+  NativeSyntheticEvent,
+  TextInputChangeEventData,
+  TouchableOpacity,
+} from 'react-native'
+import SvgUri from 'react-native-svg-uri'
+import theme from '../../theme/index'
+import * as Styled from './input.styled'
+import Search from '../../../assets/images/search.svg'
+import Close from '../../../assets/images/close.svg'
+
+type Props = {
+  defaultValue?: string
+  placeholder?: string
+  editable: boolean
+  size: keyof typeof theme.inputSizes
+  maxLength?: number
+  value?: string
+  onChange: (e: NativeSyntheticEvent<TextInputChangeEventData>) => void
+  clearButtonMode: boolean
+}
+
+export const SearchInput = ({
+  defaultValue,
+  placeholder,
+  editable,
+  size,
+  value,
+  onChange,
+  clearButtonMode,
+  maxLength,
+}: Props) => {
+  const [text, setText] = useState(value)
+  return (
+    <Styled.StyledView editable={editable}>
+      <Search
+        width={theme.searchSizes[size]}
+        height={theme.searchSizes[size]}
+      />
+      <Styled.StyledInput
+        defaultValue={defaultValue}
+        placeholder={placeholder}
+        editable={editable}
+        size={size}
```

Comment:
Missed that in the first review, but it seems that we can spread whole props obj and pass on it to the inner input